README.md

```diff
@@ -57,6 +57,19 @@ trainer = trlx.train('EleutherAI/gpt-j-6B', samples=['dolphins', 'geese'], rewar
 trainer.generate(**tokenizer('Q: Who rules the world? A:', return_tensors='pt'), do_sample=True)
 ```
 
+#### Configure Hyperparameters
+
+```python
+from trlx.data.default_configs import default_ppo_config, TrainConfig
+
+config = default_ppo_config()
+config.model.model_path = 'EleutherAI/gpt-neox-20b'
+config.train.seq_length = 32
+config.train.batch_size = 16
+
+trainer = trlx.train(config=config, reward_fn=lambda samples, **kwargs: [float(int(sample)) for sample in samples])
```

Comment:
Nit: I'm sure this example is not meant to be taken literally, but I still have some suggestions here :blush:: 
- `float(int(sample))` will throw a `ValueError`, maybe let's just take the `reward_fn` from the above
- `model_path` is updated to neox-20b, but the default ppo_config still has gpt2 tokenizer
- maybe something like `batch_size=1` and `seq_length=2048` will be more consistent with 20b model

Reply:
Ah good catch! no its a good idea to have examples copy-pastable working